ruff.toml

```diff
@@ -0,0 +1,5 @@
+line-length = 120
+
+[lint]
+select = ["C", "E", "F", "R", "W", "I", "N", "Q", "UP", "RUF", "ISC", "T20"]
+ignore = ["C9"]  # Author disagrees with forcing a specific amount of branch complexity in functions
```

Comment:
Looking at the ruff list I only see `C90` and `C4` as individual categories and `C901` as a rule, is this line actually doing something?

Reply:
Good call, I meant C901 specifically